i18n/locale_cz.xml

```diff
@@ -2,8 +2,8 @@
 <l10n>
     <translationContributors>kiki</translationContributors>
     <texts>
-        <text name="input_MA_ATTACH_VEHICLE" text="ManualAttach: Připojit/Odpojit"/>
```

Comment:
Please keep the prefix of ManualAttach, this way it's easier to detect the inputs in the inputbinding window.

Reply:
The game has sections for controls, so I think it´s not necessary anymore to have prefix, but its Your project ;-)

Reply:
Ah yeah thats right (still in FS19 mode 😄)
I will check and bulk remove them later, thanks for the PR